core/shared/src/main/scala/com/lynbrookrobotics/potassium/units/Histogram.java

```diff
@@ -0,0 +1,54 @@
+package com.lynbrookrobotics.potassium.units;
+
+import java.util.function.DoubleConsumer;
+
+/**
+ * Created by the-magical-llamicorn on 4/19/17.
```

Comment:
Please port this to Scala instead of directly copying it.
https://github.com/hi2nithya/FRC-Timing-Experiments/commit/975b2961120813a581c17532e71f56b57bf3a5a7